copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>might be clearer to replace the old file extension with the new one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Good point. Thank you. Updated it accordingly in the revised notebook.</p>